src/entities/Product.js

```diff
@@ -0,0 +1,49 @@
+/* Description: 
+Represents a product that your company sells. */
+const Entity = require('./Entity');
+class Product extends Entity {   
+constructor(productName, description, isActive) {
+    super();
+    this._name = productName ?? 'No Name provided';
+    this.description = description ?? 'No description provided';
+    this.isActive = isActive ?? true;
+}
+
+// Object getters
+
+get name() {
+    return this._name;
+}
+
+get active() {
+    return this.isActive;
+}
+
+get productDetail() {
```

Comment:
I don't think this is needed at all, let the devs decide what to do with the entity outside

Reply:
Are you saying I should also delete the isActive getter, or only the productDetail getter? I'll delete the productDetail() for now until confirmation of deleting isActive()